examples/rust/candid.did

```diff
@@ -16,10 +16,13 @@ type Utxo = record {
   confirmations: nat32;
 };
 
+type BTCCanisterInfo = record {
+  btc_address : text;
+  utxos : vec Utxo;
+  balance : Satoshi;
+  txid : text;
+  update_txid : text;
+};
+
 service: (InitPayload) -> {
-  btc_address: () -> (text) query;
-  balance: () -> (nat64);
-  get_utxos: () -> (vec Utxo);
-  send: (amount: nat64, destination: text) -> ();
```

Comment:
There are two possibilities:
1. proposing the same interfaces as before where the user can trigger functions at his will or
2. having a test code using all functions (`btc_address`, `balance`, ...)

I thought that making a test code using all functions instead of an endpoint for each function looks more straightforward.
Do you prefer to restore interfaces similar as before with five endpoints each one calling a function from the bitcoin library ?